src/diffpy/pdfgui/gui/extendedplotframe.py

```diff
@@ -29,36 +29,32 @@
 from matplotlib.font_manager import FontProperties
 import wx
 
-
 DATA_SAVE_ID  = wx.NewId()
-# NOTE: for reference, only comment out unwanted icons
+# NOTE: default toolitem at matplotlib.backend_bases
 CUSTOMIZED_TOOLITEMS = (
         ('Home', 'Reset original view', 'home', 'home'),
         ('Back', 'Back to  previous view', 'back', 'back'),
         ('Forward', 'Forward to next view', 'forward', 'forward'),
         (None, None, None, None),
         ('Pan', 'Pan axes with left mouse, zoom with right', 'move', 'pan'),
         ('Zoom', 'Zoom to rectangle', 'zoom_to_rect', 'zoom'),
-        #('Subplots', 'Configure subplots', 'subplots', 'configure_subplots'),
         (None, None, None, None),
-        #('Save', 'Save the figure', 'filesave', 'save_figure')
+        ('Export plot data', 'Export plot data to file',
+         'filesave', 'save_figure')
```

Comment:
This seems to just change the tooltip of the standard MPL "Save the figure" icon to "Export plot data", but the method name is the same - this still just saves the figure.

Actually, we need 2 save icons:
1. Save the figure - keep the standard MPL icon here
2. Export plot data - should be a new icon with its own event which eventually needs to call `ExtendedPlotFrame.savePlotData`.  See the [MPL sources](https://github.com/matplotlib/matplotlib/blob/94ed61cd7d11a5eb7a9ee13b8a1f7c7da3e986ea/lib/matplotlib/backend_bases.py#L2727-L2733) for meaning of `tooltips` entries and specification of their handler functions.  Given the existing PDFgui code, the handler function should perhaps post the `DATA_SAVE_ID` wx event.